tests/run-make/cdylib/rmake.rs

```diff
@@ -0,0 +1,39 @@
+// This test tries to check that basic cdylib libraries can be compiled and linked successfully
+// with C code, that the cdylib itself can depend on another rlib, and that the library can be built
+// with LTO.
+//
+// - `bar.rs` is a rlib
+// - `foo.rs` is a cdylib that relies on an extern crate `bar` and defines two `extern "C"`
+//   functions:
+//     - `foo()` which calls `bar::bar()`.
+//     - `bar()` which implements basic addition.
+
+// @ignore-cross-compile
```

Comment:
Problem: this needs to be

```suggestion
//@ ignore-cross-compile
```

Reply:
cc https://github.com/rust-lang/rust/issues/125813